app/uk/gov/hmrc/bindingtariffclassification/service/ReportService.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.bindingtariffclassification.service
+
+import javax.inject.Inject
+import uk.gov.hmrc.bindingtariffclassification.model.reporting.{CaseReport, ReportResult}
+import uk.gov.hmrc.bindingtariffclassification.repository.CaseRepository
+
+import scala.concurrent.Future
+
+class ReportService @Inject()(caseRepository: CaseRepository) {
```

Comment:
A general comment about our services in this app is that they seem very 'anemic' and the methods typically do nothing expect delegate to the repos. Maybe an indication that we're doing too much in the controller or repo layer

Reply:
Yep good point, I don't think there's a whole lot of business logic in this Microservice (intentionally) which usually belongs in the Service Layer. The only logic we have really is the case reference generation. Not sure what controller/repo code we could pull into the services though really, everything seems in the right place already to me